### ISSUE TITLE
openCV 4.2 build completes run main.go displays warning

### ISSUE THREAD

**Issue by K2AAE** [hybridgroup/gocv#630]
Enviroment: Jetson Nano JP4.3 had OpenCV 4.1.1 installed, followed guide and after awhile, no errors, it completed then:
go run ./cmd/version/main.go

# gocv.io/x/gocv
dnn.cpp: In function ‘cv::Mat* Net_BlobFromImage(Mat, double, Size, Scalar, bool, bool)’:
dnn.cpp:119:102: warning: passing NULL to non-pointer argument 1 of ‘cv::Scalar_<_Tp>::Scalar_(_Tp) [with _Tp = double]’ [-Wconversion-null]
         return new cv::Mat(cv::dnn::blobFromImage(*image, scalefactor, sz, NULL, swapRB, crop, ddepth));
                                                                                                      ^
gocv version: 0.22.0
opencv lib version: 4.2.0

go version go1.14 linux/arm64 (installed yesterday along with Visual Code Studio)

**Comment by PuKoren** [comment]
Do you have an exist status code? Because I also see what you posted but it is just a warning, and so far it didn't prevent my program from running

**Comment by deadprogram** [comment]
@PuKoren is exactly correct. This is only a warning message, not an error.

**Comment by karl-ford** [comment]
This might help [https://github.com/hybridgroup/gocv/issues/582#issuecomment-606016107](https://github.com/hybridgroup/gocv/issues/582#issuecomment-606016107)

**Comment by hbmask** [comment]
```
//        return new cv::Mat(cv::dnn::blobFromImage(*image, scalefactor, sz, NULL, swapRB, crop, ddepth));
        return new cv::Mat(cv::dnn::blobFromImage(*image, scalefactor, sz, cv::Scalar(), swapRB, crop, ddepth));
```

**Comment by deadprogram** [comment]
This has been released with v0.23 so now closing. Thanks!
